windows/icu-easyrpg/portfile.cmake

```diff
@@ -234,9 +287,7 @@ endif()
 # Generates warnings about missing pdbs for icudt.dll
 # This is expected because ICU database contains no executable code
 vcpkg_copy_pdbs()
+vcpkg_fixup_pkgconfig(SYSTEM_LIBRARIES pthread m)
 
 # Handle copyright
-# EASYRPG CUSTOM
-file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg/LICENSE ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg/copyright)
-# EASYRPG END
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

Comment:
Not sure why here a customization was needed, it should be the same with this upstream line.

Reply:
There is a sanity check that makes the build fail when the license file is missing. The $PORT hopefully contains the correct content